src/view_control.cpp

```diff
@@ -1877,16 +1948,21 @@ void View::DrawPedal(DeviceContext *dc, Pedal *pedal, Measure *measure, System *
     }
     str.push_back(code);
 
-    for (staffIter = staffList.begin(); staffIter != staffList.end(); ++staffIter) {
-        if (!system->SetCurrentFloatingPositioner((*staffIter)->GetN(), pedal, pedal->GetStart(), *staffIter)) {
-            continue;
-        }
-        // Basic method that use bounding box
-        int y = pedal->GetDrawingY();
+    dc->StartGraphic(pedal, "", pedal->GetUuid());
 
-        dc->SetFont(m_doc->GetDrawingSmuflFont((*staffIter)->m_drawingStaffSize, false));
-        DrawSmuflString(dc, x, y, str, alignment, (*staffIter)->m_drawingStaffSize);
-        dc->ResetFont();
+    // Don't draw a symbol, if it's a line
```

Comment:
Confusing, say // Draw a symbol if it is not a line
But there is something not very clear here: `str` is modified above but then not used if is it a line. I think the logic can be improved.